assets/src/dashboard/app/views/shared/pageHeading.js

```diff
@@ -18,100 +18,74 @@
   display: flex;
-  white-space: nowrap;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 16px;
+  padding-top: 48px;
+  padding-bottom: 24px;
+  border-bottom: 1px solid ${({ theme }) => theme.colors.divider.secondary};
 `;
 
-const Content = styled.div`
+const StyledHeadline = styled(Display)`
   display: flex;
-  align-items: ${({ centerContent }) =>
-    centerContent ? 'center' : 'flex-end'};
-  height: 100%;
-`;
-
-const SearchContainer = styled.div`
-  display: inline-block;
-  vertical-align: baseline;
-  position: relative;
-  width: 100%;
-  height: 29px;
-  @media ${({ theme }) => theme.DEPRECATED_THEME.breakpoint.smallDisplayPhone} {
-    left: ${({ theme }) =>
-      `${theme.DEPRECATED_THEME.standardViewContentGutter.min}px`};
-    max-width: 100%;
-    justify-content: flex-start;
-  }
+  align-items: center;
+  margin-right: 8px;
+  white-space: nowrap;
 `;
 
-const SearchInner = styled.div`
-  position: absolute;
-  top: 0;
-  right: 0;
-  width: min(${DASHBOARD_LEFT_NAV_WIDTH}px, 100%);
+const HeadlineFilters = styled.div`
   display: flex;
-  justify-content: flex-end;
-`;
-
-const HeadingBodyWrapper = styled(StandardViewContentGutter)`
-  display: grid;
-  grid-template-columns: 25% 50% 1fr;
   align-items: center;
-  margin-top: 36px;
-  padding-bottom: 24px;
-  border-bottom: ${({ theme }) =>
-    theme.DEPRECATED_THEME.subNavigationBar.border};
+  margin: 0 auto;
 `;
 
-export const HeaderToggleButtonContainer = styled.div`
-  display: block;
-  flex: 1;
-  height: 65%;
+const HeaderSearch = styled.div`
+  width: 208px;
+  max-width: 208px;
+  margin: auto 0;
 `;
 
 const PageHeading = ({
   children,
-  defaultTitle,
+  heading,
   searchPlaceholder,
-  centerContent = false,
-  stories = [],
-  showTypeahead = true,
-  handleTypeaheadChange,
-  typeaheadValue = '',
+  searchOptions = [],
+  showSearch,
+  handleSearchChange,
+  searchValue = '',
 }) => {
   return (
-    <HeadingBodyWrapper>
+    <HeadingContainer>
       <StyledHeadline
         as="h2"
-        size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_LARGE}
+        size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
       >
         <NavMenuButton showOnlyOnSmallViewport />
-        {defaultTitle}
+        {heading}
       </StyledHeadline>
-      <Content centerContent={centerContent}>{children}</Content>
-      {showTypeahead && (
-        <SearchContainer>
-          <SearchInner>
-            <TypeaheadSearch
-              placeholder={searchPlaceholder}
-              currentValue={typeaheadValue}
-              stories={stories}
-              handleChange={handleTypeaheadChange}
-            />
-          </SearchInner>
-        </SearchContainer>
+      {children && <HeadlineFilters>{children}</HeadlineFilters>}
+      {showSearch && (
+        <HeaderSearch>
+          <Search
+            placeholder={searchPlaceholder}
+            selectedValue={{ label: searchValue, value: searchValue }}
```

Comment:
This is for the new search and what it expects to manage. Right now our typed input is also the value, but there's cases or potential for update where the value of the search isn't necessarily the string visible to the user.